src/qmflows/parsers/cp2KParser.py

```diff
@@ -353,17 +353,26 @@ def split_unrestricted_log_file(path: PathLike) -> Tuple[Path, Path]:
     """Split the log file into alpha and beta molecular orbitals."""
     root, file_name = os.path.split(path)
     cmd = f'csplit -f coeffs -n 1 {file_name} "/HOMO-LUMO/+2"'
-    subprocess.call(cmd, shell=True, stdout=subprocess.DEVNULL,
-                    stderr=subprocess.DEVNULL, cwd=root)
+    subprocess.check_output(cmd, shell=True, stdout=subprocess.DEVNULL, cwd=root)
 
-    alphas = Path(root, "alphas.log")
-    betas = Path(root, "betas.log")
+    # Check that the files exists
+    predicate = all(Path(f).exists() for f in ('coeffs0', 'coeffs1'))
+    if not predicate:
+        msg = "There is a problem splitting the coefficients in alpha and beta components!"
+        raise RuntimeError(msg)
```

Comment:
```suggestion
```
Whoops, forgot to include the leftovers in previous suggestion.

Reply:
no problem :) I fix it!